cpp/server/ct-server.cc

```diff
@@ -222,10 +231,30 @@ int main(int argc, char* argv[]) {
   // For now, run with a dedicated thread pool as the executor for our
   // consistent store to avoid the possibility of DoS through thread starvation
   // via HTTP.
-  ThreadPool etcd_pool(2);
-  EtcdConsistentStore<LoggedCertificate> consistent_store(&etcd_pool,
+  ThreadPool internal_pool(4);
```

Comment:
I don't feel that strongly about it either, it just seems like a magic number lying around, and figured the automatic default is probably just as good as our guessing, but not having pages of output for ```info thread``` is a decent argument, IMHO.